apps/prairielearn/src/ee/pages/lti13Instance/lti13Instance.ts

```diff
@@ -39,7 +39,7 @@ const ltiConfig = {
             placement: 'course_navigation',
             default: 'disabled',
             message_type: 'LtiResourceLinkRequest',
-            target_link_url: 'replace',
+            target_link_uri: 'replace',
```

Comment:
Ah, was this just a typo before? Is there somewhere in the LTI 1.3 spec that says "replace" is a valid value? Or were you using this to mean "replace me later, I'm a placeholder"?

Reply:
I may have answered my own question: I guess that's what line 76 is doing? That was not immediately obvious to me.

Reply:
url -> uri

Reply:
Thanks for confirming. If you'd like, consider adding a comment stating that this value will be replaced.

Reply:
I changed the placeholder text to be more self documenting in 8e2489f1

Reply:
That's much clearer, thank you!